clang/lib/CIR/Lowering/DirectToLLVM/LowerToLLVM.cpp

```diff
@@ -949,8 +949,7 @@ mlir::LogicalResult CIRToLLVMVTTAddrPointOpLowering::matchAndRewrite(
 
   if (op.getSymAddr()) {
     if (op.getOffset() == 0) {
-      rewriter.replaceAllUsesWith(op, llvmAddr);
-      rewriter.eraseOp(op);
+      rewriter.replaceOp(op, {llvmAddr});
```

Comment:
Updated here because `replaceAllUsesWith` would eagerly replace the uses with LLVM pointer values instead of recording the replacement and applying it later after conversion completes. This would cause `cir::LoadOp::getAddr` to fail since the `addr` parameter is replaced eagerly by a `!llvm.ptr` value.